.github/workflows/deploy-prod.yml

```diff
@@ -32,7 +32,7 @@ jobs:
     timeout-minutes: 60
     strategy:
       matrix:
-        node-version: [16.20.0]
+        node-version: [18.19.0]
```

Comment:
Note that these are not actually used anywhere. I've removed these definitions in 1.4.0

Reply:
@rikukissa should I revert this, then?

Reply:
@tahmidrahman-dsi I think it's fine to keep unless you want to remove the whole `matrix` block